### ISSUE TITLE
Deployment is not supported by Polaris plugin

### ISSUE THREAD

**Issue by mladedav** [aquasecurity/starboard#697]
**What steps did you take and what happened:**

I tried to call scan with polaris on a deployment as shown in the documentation here https://aquasecurity.github.io/starboard/v0.10.3/cli/getting-started/
```
$ starboard scan configauditreports deployment/deployment-name
error: kind Deployment is not supported by Polaris plugin
```

**What did you expect to happen:**

Get a config audit report from Polaris

**Anything else you would like to add:**

[Miscellaneous information that will assist in solving the issue.]

**Environment:**

- Starboard version (use `starboard version`):
Starboard Version: {Version:0.12.0-rc2 Commit:3a5723d63d2fed1729d0bf5e748d8dce05412c5b Date:2021-08-27T12:21:53Z}
- Kubernetes version (use `kubectl version`):
Client Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.0", GitCommit:"af46c47ce925f4c4ad5cc8d1fca46c7b77d13b38", GitTreeState:"clean", BuildDate:"2020-12-08T17:59:43Z", GoVersion:"go1.15.5", Compiler:"gc", Platform:"windows/amd64"}
Server Version: version.Info{Major:"1", Minor:"21", GitVersion:"v1.21.2", GitCommit:"bd9d9ae2719e67a582f48c5dc2f81e87fe1deb8a", GitTreeState:"clean", BuildDate:"2021-08-16T22:02:14Z", GoVersion:"go1.16.5", Compiler:"gc", Platform:"linux/amd64"}
- OS (macOS 10.15, Windows 10, Ubuntu 19.10 etc): Windows 10 (native, no WSL)

**Comment by danielpacak** [comment]
👋 @mladedav It's been fixed in 531292e and pre-released in [v0.12.0-rc3](https://github.com/aquasecurity/starboard/releases/tag/v0.12.0-rc3). Please reopen the issue if the problem persists.

**Comment by mladedav** [comment]
Thank you, you're right. Sorry, I wasn't able to find the original issue.
